deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv

```diff
@@ -260,21 +228,1172 @@ registration-processor-default.properties,GETVIDSBYUIN
+pre-registration-default.properties,mosip.batch.token.authmanager.password
+pre-registration-default.properties,batch.appointment.cancel
+pre-registration-default.properties,masterdata.service.env
+pre-registration-default.properties,holiday.exceptional.url
+pre-registration-default.properties,working.day.url
+pre-registration-default.properties,mosip.kernel.masterdata.validdoc.rest.uri
+pre-registration-default.properties,notification.service.env
+pre-registration-default.properties,email.service.env
+pre-registration-default.properties,sms.service.env
+pre-registration-default.properties,audit.service.env
+pre-registration-default.properties,crypto.service.env
+pre-registration-default.properties,secretKey
+pre-registration-default.properties,kernel.auth.env
+pre-registration-default.properties,kernel.prid.env
+pre-registration-default.properties,mosip.preregistration.id-schema
+pre-registration-default.properties,mosip.base.url
+pre-registration-default.properties,ui.config.params
+pre-registration-default.properties,preregistration.ui.version
+pre-registration-default.properties,mosip.preregistration.captcha.secretkey
+print-default.properties,mosip.event.secret
+print-default.properties,PDFSIGN
+print-default.properties,registration.processor.identityjson
+print-default.properties,mosip.print.prependThumbprint
+registration-default.properties,mosip.registration.consent_fra
+registration-default.properties,mosip.mdm.enabled
+registration-default.properties,mosip.kernel.transliteration.arabic-language-code
+registration-default.properties,mosip.registration.reset_password_url
+registration-processor-default.properties,registration.processor.zone
+registration-processor-default.properties,cluster.manager.file.name
+registration-processor-default.properties,camel.secure.active.flows.file.names
+registration-processor-default.properties,token.request.secretKey
+registration-processor-default.properties,INTERNALAUTH
+registration-processor-default.properties,DEVICEVALIDATEHISTORY
+registration-processor-default.properties,IDREPOSITORY
+registration-processor-default.properties,IDREPOGETIDBYUIN
+registration-processor-default.properties,UINGENERATOR
+registration-processor-default.properties,RETRIEVEIDENTITYFROMRID
+registration-processor-default.properties,RETRIEVEIDENTITY
+registration-processor-default.properties,CREATEVID
+registration-processor-default.properties,GETUINBYVID
+registration-processor-default.properties,CREDENTIALREQUEST
+registration-processor-default.properties,GETVIDSBYUIN
+registration-processor-default.properties,ENCRYPTURL
+registration-processor-default.properties,ENCRYPTIONSERVICE
+registration-processor-default.properties,DIGITALSIGNATURE
+registration-processor-default.properties,CRYPTOMANAGERDECRYPT
+registration-processor-default.properties,SMSNOTIFIER
+registration-processor-default.properties,EMAILNOTIFIER
+registration-processor-default.properties,PMS
```

Comment:
there is something wrong here, it show too many new lines added, but we are only deleting entries from all 3 files.
In this file we are suppose to delete 42 entries, that should be the only change

Reply:
updated sir.